cypress/integration/branch/sensor-connection.test.js

```diff
@@ -86,7 +86,6 @@ context('Collecting Data from 1 sensor',()=>{
     it('verify Start button is disabled when data collection is started',()=>{
         let duration="30";
         workspace.getNewRunButton().click()
-        workspace.discardData()
```

Comment:
Did you mean for this to be in the PR as it doesn't match the description of the commit.

Reply:
I did mean for it to be in there - I had to remove this line from the other PR too, because we are no longer showing the warning every time a user clicks "New Run".